airbyte-integrations/bases/connector-acceptance-test/build.gradle

```diff
@@ -1,3 +1,5 @@
+// The airbyte-python is commented out because connector-acceptance-test are now installed with poetry and use Python 3.10
+// The python Gradle plugin can't install connector-acceptance-test in this situation.
 plugins {
     id 'airbyte-docker'
```

Comment:
`airbyte-docker` is kept to support `/legacy-publish` until `airbyte-ci cat publish` command exists.